stripper/ze_doom3_v2.cfg

```diff
@@ -610,23 +680,524 @@ modify:
+modify:
+{
+	match:
+	{
+		"classname" "func_button"
+		"hammerid" "419197"
+	}
+	insert:
+	{
+		"OnPressed" "cmdCommandsay *** Teleporter activating in 15s ***261"
+	}
+}
+modify:
+{
+	match:
+	{
+		"classname" "logic_relay"
+		"targetname" "stage3_hell_die"
+	}
+	insert:
+	{
+		"OnTrigger" "CmdCommandsay ** Lift rising in 30s **751"
+		"OnTrigger" "CmdCommandsay ** Lift rising **1051"
+	}
+}
+modify:
+{
+	match:
+	{
+		"classname" "trigger_teleport"
+		"hammerid" "2673"
+	}
+	insert:
+	{
+		"OnStartTouch" "CmdCommandsay ** ZMs tp in 10s **01"
+	}
+}
+modify:
+{
+	match:
+	{
+		"classname" "trigger_once"
+		"hammerid" "3634"
+	}
+	insert:
+	{
+		"OnStartTouch" "CmdCommandsay ** ZMs tp in 10s **01"
+	}
+}
+modify:
+{
+	match:
+	{
+		"classname" "trigger_once"
+		"hammerid" "4060"
+	}
+	insert:
+	{
+		"OnStartTouch" "CmdCommandsay ** ZMs tp in 10s **01"
+	}
+}
+; Stage 3 lava lake should only tp zms
+modify:
+{
+	match:
+	{
+		"targetname" "stage3_hell_tp_lava"
+		"classname" "trigger_teleport"
+	}
+	replace:
+	{
+		"filtername" "zm"
+	}
+}
+; Surround stage 3 ending lava lake with walls to stop boosted zms from landing outside the map
+add:
+{
+	"targetname" "strp_wall_1"
+	"classname" "func_wall_toggle"
+	"rendermode" "10"
+	"origin" "-10848 -5160 6459"
+}
+modify:
+{
+	match:
+	{
+		"classname" "logic_auto"
+		"hammerid" "135137"
+	}
+	insert:
+	{
+		"OnMapSpawn" "strp_wall_1AddOutputsolid 20.51"
+		"OnMapSpawn" "strp_wall_1AddOutputmins -4576 -680 -269311"
+		"OnMapSpawn" "strp_wall_1AddOutputmaxs 4576 680 269311"
+	}
+}
+add:
+{
+	"targetname" "strp_wall_2"
+	"classname" "func_wall_toggle"
 
-		"OnMapSpawn" "strp_tp_stg2_gapAddOutputsolid 20.51"
-		"OnMapSpawn" "strp_tp_stg2_gapAddOutputmins -2078 -718 -111.511"
-		"OnMapSpawn" "strp_tp_stg2_gapAddOutputmaxs 2078 718 111.511"
+	"origin" "-10976 1280 6496"
+}
+modify:
+{
+	match:
+	{
+		"classname" "logic_auto"
+		"hammerid" "135137"
+	}
+	insert:
+	{
+		"OnMapSpawn" "strp_wall_2AddOutputsolid 20.51"
+		"OnMapSpawn" "strp_wall_2AddOutputmins -4608 -464 -269311"
+		"OnMapSpawn" "strp_wall_2AddOutputmaxs 4608 464 269311"
+	}
+}
+add:
+{
+	"targetname" "strp_wall_3"
+	"classname" "func_wall_toggle"
+	"rendermode" "10"
+	"origin" "-6016 -2024 6400"
+}
+modify:
+{
+	match:
+	{
+		"classname" "logic_auto"
+		"hammerid" "135137"
+	}
+	insert:
+	{
+		"OnMapSpawn" "strp_wall_3AddOutputsolid 20.51"
+		"OnMapSpawn" "strp_wall_3AddOutputmins -400 -3816 -262411"
+		"OnMapSpawn" "strp_wall_3AddOutputmaxs 400 3816 262411"
+	}
+}
+add:
+{
+	"targetname" "strp_wall_4"
+	"classname" "func_wall_toggle"
+	"rendermode" "10"
+	"origin" "-15656.4 -2062.58 6427"
+}
+modify:
+{
+	match:
+	{
+		"classname" "logic_auto"
+		"hammerid" "135137"
+	}
+	insert:
+	{
+		"OnMapSpawn" "strp_wall_4AddOutputsolid 20.51"
+		"OnMapSpawn" "strp_wall_4AddOutputmins -400 -3816 -262411"
+		"OnMapSpawn" "strp_wall_4AddOutputmaxs 400 3816 262411"
+	}
+}
```

Comment:
Custom sizing only works for triggers, solid brushes must use an existing brush model in the map.